canonical-web-and-design/www.ubuntu.com
# Change urls for case studies

## Done

- Change urls for case studies
- Add redirect for /engage/autonomous-more-profitable-farming-using-AI-and-lightweight-robots
- Add redirect for /engage/North-Americas-first-autonomous-runway-snowplough-case-study


## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/engage/small-robot-company](http://0.0.0.0:8001/engage/small-robot-company) and [/engage/Northstar](http://0.0.0.0:8001/engage/Northstar)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure the urls are changed as requested [here](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5251)
- Make sure there is a redirect for /engage/autonomous-more-profitable-farming-using-AI-and-lightweight-robots
- Make sure there is a redirect for /engage/North-Americas-first-autonomous-runway-snowplough-case-study


## Issue / Card

Fixes #5251
